src/client.ts

```diff
@@ -464,14 +464,11 @@ export function createClient<
     connectionAckWaitTimeout = 0,
     retryAttempts = 5,
     retryWait = async function randomisedExponentialBackoff(retries) {
-      let retryDelay = 1000; // start with 1s delay
-      for (let i = 0; i < retries; i++) {
-        retryDelay *= 2;
-      }
+      const retryDelaySeconds = Math.pow(2, retries);
```

Comment:
If it's meant to be "seconds" (because of the suffix), then:
```suggestion
      const retryDelaySeconds = Math.pow(2, retries) * 1000;
```
Right?

Reply:
Oof, right. Great!